### ISSUE TITLE
Xamarin & ScanFlow: Is it possible to require a minimum document size for successful scan?

### ISSUE THREAD

**Issue by MihaMarkic** [thegrizzlylabs/geniusscan-sdk-demo#68]
By default document is fine as long as it is recognized, no matter it's relative size to camera size. i.e. if user takes a photo from a distance it will still produce a valid result, regardless of documents size in pixels.
I'd like to enforce a minimum ratio (or area) that document has to fill to product an image with enough pixels.
Basically I'd pass a percentage factor and UI would refuse to scan smaller documents (perhaps with a feedback of different color).
Is it possible with ScanFlow? If not, with custom UI?

**Comment by MihaMarkic** [comment]
Hi, got it. But it'd be even better if I could show to user feedback in real time, i.e. by using a different overlay color. That way user doesn't have to wait for document detection and can adjust position immediately.
Do you have any timeline for Scan flow implementation?

**Comment by guillaume-tgl** [comment]
You're right, that's not ideal. We'll try to to either provide this in the ScanFlow or make it possible in the Core module. What's your need exactly? You'd like each scan to be at least a certain resolution?

**Comment by MihaMarkic** [comment]
>What's your need exactly? You'd like each scan to be at least a certain resolution?

Exactly. Because of OCR.
